.github/ISSUE_TEMPLATE/template.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Template
+about: 'Standardize issues. '
+title: ''
```

Comment:
What is the difference between name and title?

Reply:
I believe title is the specific title for each issue, whilst name is just for the template itself.